package.json

```diff
@@ -12,12 +12,36 @@
     "url": "https://github.com/stoplightio/http-spec"
   },
   "license": "Apache-2.0",
-  "main": "src/index.ts",
   "files": [
     "**/*"
   ],
   "engines": {
-    "node": ">=10"
+    "node": "^12.20 || >=14.13"
```

Comment:
Should we take this opportunity to drop node 12? 🤓

Reply:
why not :D
```suggestion
    "node": ">=14.13"
```